src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -593,7 +659,19 @@ synchronized public Version getBridgeVersion(String bridgeJid)
          * If not set we consider it highly occupied,
          * because no stats we have been fetched so far.
          */
-        private int videoStreamCount = Integer.MAX_VALUE;
+        private Integer videoStreamCount = null;
```

Comment:
No and it doesn't make sense anymore to have nullable type, so I'm changing it to int. Thanks !